docs/SRS-Volere/SRS.tex

```diff
@@ -154,19 +154,36 @@ \subsection{Accessibility Requirements}
 
 \section{Performance Requirements}
 \subsection{Speed and Latency Requirements}
-\lips
+\begin{itemize}
+  \item \textbf{SLR1}: The app should take on average 1 second to complete backend tasks and at worst should never
+  exceed 12 seconds.
+  \item \textbf{SLR2}: The navigation of the app should be smooth, taking no more than 1 second render and handle page changes.
```

Comment:
should you add like what happens if these aren't met?

Reply:
I think if they aren't met, we change the requirements and talk to stakeholders. We shouldn't be putting contingencies as part of the requirements, or at least, not in this context.